frontend/src/components/Search.js

```diff
@@ -44,7 +44,7 @@ const Search = () => {
   }
 
   return (
-    <div className="search-box" onClick={notifyMessage}>
+    <div className="search-box" onClick={notifyMessage || null}>
```

Comment:
What does this change do?

Reply:
NotifyMessage is function executing only when user try to search something when actually can't.
For example we are on edit-post route(so user can't search there, he must leave first), message is executing, everything is okey.
But where he is searching on "allowed route" for it, function is executing but in real customHook doesn't return that one 

`
Warning: Expected `onClick` listener to be a function, instead got `false`.
If you used to conditionally omit it with onClick={condition && value}, pass onClick={condition ? value : undefined} instead.
`
 I think this just hard code instance, should i use in customHook options that can return "null" instead of in component?